<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeReminderWindow</class>
 <widget class="QMainWindow" name="TimeReminderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="labelPic">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelEvent">
                <property name="styleSheet">
                 <string notr="true">font: 25 36pt &quot;Microsoft YaHei&quot;;
qproperty-alignment: 'AlignCenter';</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelRemarks">
                <property name="styleSheet">
                 <string notr="true">font: 25 36pt &quot;Microsoft YaHei&quot;;
qproperty-alignment: 'AlignCenter';</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelTimeStr">
                <property name="styleSheet">
                 <string notr="true">font: 25 36pt &quot;Microsoft YaHei&quot;;
qproperty-alignment: 'AlignCenter';</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
